system-contracts/contracts/Constants.sol

```diff
@@ -122,3 +129,6 @@ uint256 constant STATE_DIFF_DERIVED_KEY_OFFSET = 52;
 uint256 constant STATE_DIFF_ENUM_INDEX_OFFSET = 84;
 /// @dev The offset of the final value in a storage diff.
 uint256 constant STATE_DIFF_FINAL_VALUE_OFFSET = 124;
+
+/// @dev Total number of bytes in a blob. Blob = 4096 field elements * 32 bytes per field element/
```

Comment:
please double check with the crypto team. Each field element is less than 32 bytes, so 4096 * 31 may be more correct

Reply:
@koloz193 Does anyone from the crypto team approve 31 bytes? It just looks weird to me that EIP states to use 32 bytes for hashing

Reply:
i asked for the crypto team to review the PR, per @jules: 

> we do chunks of 31 bytes so that we always fit in a field element.we take a small size hit to ensure for performance that we don’t have to do assertions and checks for blob elements

Reply:
<img width="1230" alt="image" src="https://github.com/matter-labs/era-contracts/assets/41153528/e23e83aa-20b1-495e-a827-f77230d69a5f">